scripts/prepublish

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+set -euf -o pipefail
+
+node_modules/.bin/sanctuary-prepublish "$@"
+
+rm -rf node_modules
+npm install
+mkdir -p -- dist
+scripts/bundle >dist/bundle.js
```

Comment:
I updated `scripts/bundle` to write to standard output, to make it apparent that `mkdir -p` must be used.